operations/rategroups.md

```diff
@@ -45,44 +46,44 @@ Returns all rate groups, filtered by unique identifiers and other filters. Note
-| `ServiceIds` | array of string | optional, max 1000 items | Unique identifiers of the [Service](services.md#service). Required if `RateGroupIds` filter is not provided. |
-| `ExternalIdentifiers` | string | optional, max 1000 items | Identifiers of [Rate group](#rate-group) from external systems. |
-| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Rate group](#rate-group) was updated. |
-| `ActivityStates` | array of string [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted or both records. |
-| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of rate groups returned. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the Enterprises. If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `RateGroupIds` | array of string | optional, max 1000 items | Unique identifiers of the [Rate group](rategroups.md#rate-group). Required if ServiceIds filter is not provided. |
+| `ServiceIds` | array of string | optional, max 1000 items | Unique identifiers of the [Service](services.md#service). Required if RateGroupIds filter is not provided. |
+| `ExternalIdentifiers` | array of string | optional, max 1000 items | Identifiers of [Rate group](rategroups.md#rate-group) from external systems. |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months |  |
+| `ActivityStates` | array of [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted or both records. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
 
 ### Response
 
 ```javascript
 {
-    "RateGroups": [
-        {
-            "Id": "6b3f718a-b537-45b0-a8ee-d30897723834",
-            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
-            "IsActive": true,
-            "CreatedUtc": "2023-01-11T13:32:32Z",
-            "UpdatedUtc": "2023-01-13T15:27:28Z",
-            "Ordering": 0,
-            "Names": {
-                "en-US": "Default group"
-            },
-            "ShortNames": {
-                "en-US": "Default group"
-            },
-            "Descriptions": {},
-            "ExternalIdetnfier": "RG-001"
-        }
-    ],
-    "Cursor": "6b3f718a-b537-45b0-a8ee-d30897723834"
+  "RateGroups": [
+    {
+      "Id": "6b3f718a-b537-45b0-a8ee-d30897723834",
+      "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+      "IsActive": true,
+      "CreatedUtc": "2023-01-11T13:32:32Z",
+      "UpdatedUtc": "2023-01-13T15:27:28Z",
+      "Ordering": 0,
+      "Names": {
+        "en-US": "Default group"
+      },
+      "ShortNames": {
+        "en-US": "Default group"
+      },
+      "Descriptions": {},
+      "ExternalIdetnfier": "RG-001"
```

Comment:
typo

Reply:
Shouldn't this be automatically generated?

Reply:
I didn't put example in the code so it took what was already in the docs 👍 I'll put example in the code so it will pick it up